redaxo/src/addons/be_style/plugins/customizer/pages/system.customizer.php

```diff
@@ -119,7 +119,7 @@
 
 $n = [];
 $n['label'] = '<label for="customizer-labelcolor">' . rex_i18n::msg('customizer_labelcolor') . '</label>';
-$n['field'] = '<input class="form-control" id="customizer-labelcolor" type="text" name="settings[labelcolor]" value="' . htmlspecialchars($config['labelcolor']) . '" />';
+$n['field'] = '<input class="form-control minicolors" id="customizer-labelcolor" type="text" name="settings[labelcolor]" value="' . htmlspecialchars($config['labelcolor']) . '" />';
```

Comment:
"minicolors" erzeugt also einen colorpicker?

Reply:
genau, wenn, man das FOR addon "UI Tools" installiert hat.